### DISCUSSION TITLE
Goal 13: Take urgent action to combat climate change and its impacts

### DISCUSSION THREAD

**Discussion by HSCunha** [Project-Resilience/platform#35] [General]
Chat about anything related to Goal 13: [Take urgent action to combat climate change and its impacts](https://github.com/Project-Resilience/platform/blob/main/goals/goal_13.md)

**Comment by danyoungday** [comment]
# EX-ACT Tool by the FAO
"The Environmental eXternalities ACcounting Tool (EX-ACT, previously EX-Ante Carbon balance Tool), provides users a consistent way of estimating and tracking the outcomes of agricultural interventions on GHG emissions." Users can create an account and try it out here: [https://exact.apps.fao.org/]( https://exact.apps.fao.org/).

## Data
### Context
- Country
- Climate type
- Moisture
- Project duration, implementation phase
Users can also choose the method with which emissions are calculated (AR4, AR5, AR6 reports)

### Actions
There are many different categories of actions that can be taken. Each action module has several components within it.
- Afforestation and forest management
- Flooded rice management
- Perennial cropland and agroforestry management
- Annual cropland management
- Coastal wetland management
- Infrastructure or settlement creation
- Set land aside
- Other land
- Livestock management
- Water body management
- Marine and inland fisheries
- Aquaculture
- Use of agricultural inputs, energy or irrigation
- Downstream value chain intervention
- Grassland management

### Outcomes
Outcomes are computed for each action taken as well as aggregated into a comprehensive result
- Total GHG fluxes (tonnes of CO2 equivalent)
    - With and without intervention, many actions have fields such as "what would have happened if you didn't take this intervention"
- Carbon balance by specific GHG (CH4, N2O, CO2, Other)
- Mitigation cost

Furthermore, other cost outcomes can be computed directly from the actions taken.
